File: KtMapper/src/main/java/cn/misection/ktrelation/service/RelationService.java
package cn.misection.ktrelation.service;

import java.util.List;
import java.util.Map;

import cn.misection.relation.entity.PersonRelationPair;

/**
 * @author Military Intelligence 6 root
 * @version 1.0.0
 * @ClassName RelationDAO
 * @Description TODO
 * @CreateTime 2022年02月25日 13:20:00
 */
public interface RelationService {

  List<PersonRelationPair> getRelation();

  Map<String, String> getRelationMap();
}

File: KtMapper/src/main/kotlin/cn/misection/relation/service/RelationService.kt
package cn.misection.relation.service

import cn.misection.relation.entity.PersonRelationPair

/**
 * @ClassName RelationService
 * @author Military Intelligence 6 root
 * @version 1.0.0
 * @Description TODO
 * @CreateTime 2022年02月26日 15:49:00
 */
interface RelationService {

  fun getRelation(): List<PersonRelationPair>

  fun getRelationMap(): Map<String, String>
}
